source/_components/tradfri.markdown

```diff
@@ -38,3 +38,7 @@ Configuration variables:
 <p class='note'>
 Do not use the `api_key` variable. The key is only needed once at initial setup.
 </p>
+
+<p class='note'>
+Please make sure you have autoconf installed (```sudo apt-get install autoconf```) and setuptools (```sudo pip3 install setuptools```) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices.
```

Comment:
Python packages should be installed in the environment running home assistant, so use of `sudo` is environment specific and not generally recommended.